src/modules/handlers/screen/poster/index.ts

```diff
@@ -1,3 +1,3 @@
 // eslint-disable-next-line import/no-cycle -- TODO fix cyclic dependency
-export { PosterScreenController } from './poster-screen-controller';
-export { default as PosterScreenHandler } from './poster-screen-handler';
+export { BasePosterScreenController } from './base-poster-screen-controller';
```

Comment:
What I mean is that we should not include these base classes in `index.ts`, because this file contains exports of the folder (so you can import classes from the same module nicely in a single line, i.e. `import { PosterScreenController } from '../handlers/screen/poster`). It should therefore also include all classes/consts that you wish to expose.

Reply:
Fixed, the imports should be cleaned up now, it seemed this index was not used anywhere so I removed it. the index under screen is already aggregating all the handlers, and the controllers are not imported afaik